app/connectors/CustomsDeclareExportsMovementsConnector.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 HM Revenue & Customs
+ * Copyright 2020 HM Revenue & Customs
```

Comment:
Please rebase your branch to the master. As I remember @MaciejRewera updated all the headers